tf6to5server/doc.go

```diff
@@ -0,0 +1,12 @@
+// Translate a protocol version 6 provider server into protocol version 5.
+//
+// Supported protocol version 6 provider servers include any which implement
+// the github.com/hashicorp/terraform-plugin-go/tfprotov6.ProviderServer
+// interface, such as:
+//
+// - github.com/hashicorp/terraform-plugin-framework
+// - github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server
+// - github.com/hashicorp/terraform-plugin-mux/tf5to6server
```

Comment:
Shouldn't this be `github.com/hashicorp/terraform-plugin-mux/tf6to5server`?

Reply:
`tf5to6server` implements the `tfprotov6.ProviderServer` interface 😉 

https://github.com/hashicorp/terraform-plugin-mux/blob/3b76c74adaef3b1256c9aabee746aa746a67b2d3/tf5to6server/tf5to6server.go#L26

Although now that I'm re-reading this, we probably shouldn't mention it because it'd be weird to use _both_ `tf6to5server` and `tf5to6server` since you can just serve the protocol version 5 server. What I meant to document was `tf6muxserver`. 😅 

```suggestion
// - github.com/hashicorp/terraform-plugin-mux/tf6muxserver
```